practice/2_python_part_2/task_input_output.py

```diff
@@ -15,9 +15,21 @@
     No numbers entered
 
 """
-
+import sys
 
 def read_numbers(n: int) -> str:
-    ...
+    arguments = sys.argv[1:]
```

Comment:
Task definition says: `Write function which reads a number from input nth times`.
means you need to call the `input` function